fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/AvatarGroup.kt

```diff
@@ -81,29 +81,38 @@ fun AvatarGroup(
     Layout(modifier = modifier
         .padding(8.dp)
         .then(semanticModifier), content = {
-        for (i in 0 until visibleAvatar) {
-            val person = group.members[i]
-
-            var paddingModifier: Modifier = Modifier
-            if (style == AvatarGroupStyle.Pile && person.isActive) {
-                val padding = token.pilePadding(avatarGroupInfo)
-                paddingModifier = paddingModifier.padding(start = padding, end = padding)
-            }
-
-            Avatar(
-                person,
-                modifier = paddingModifier,
+        if (style == AvatarGroupStyle.Pie && group.members.size > 1) {
+            AvatarPie(
+                group = group,
                 size = size,
-                enableActivityRings = true,
-                enablePresence = enablePresence,
-                avatarToken = avatarToken
-            )
-        }
-        if (group.members.size > visibleAvatar || group.members.isEmpty()) {
-            Avatar(
-                group.members.size - visibleAvatar, size = size,
-                enableActivityRings = true, avatarToken = avatarToken
+                noOfVisibleAvatars = visibleAvatar,
+                avatarTokens = avatarToken
             )
+        } else {
+            for (i in 0 until visibleAvatar) {
+                val person = group.members[i]
```

Comment:
is ther any chance for group.members be an emptylist?

Reply:
i have added check here that group size should be greater than 0